src/tests/bulkConcepts/reducer/index.test.js

```diff
@@ -149,4 +153,30 @@ describe('Test suite for bulkConcepts reducer', () => {
       preview: action.payload,
     });
   });
+  it('should handle GET_QUESTION_ANSWERS', () => {
```

Comment:
What does "should handle" mean?

Reply:
I have omitted the multiple instances of `Should handle`